docs/user/reference/metrics/counter.md

```diff
@@ -123,8 +124,8 @@ Glean.controls.refreshPressed.add(5);
 
 ### `testGetValue`
 
-Gets the recorded value for a given counter metric.  
-Returns the count if data is stored.  
+Gets the recorded value for a given counter metric.
+Returns the count if data is stored.
```

Comment:
Note that the double-spacing at line-end here was load-bearing: It enforces line breaks in the generated HTML.